owner/manager.go

```diff
@@ -32,88 +30,79 @@ import (
 	"google.golang.org/grpc"
 )
 
-// OwnerManager is used to campaign the owner and manage the owner information.
-type OwnerManager interface {
-	// ID returns the ID of DDL.
+// Manager is used to campaign the owner and manage the owner information.
```

Comment:
Because the package's name is `owner`, so it will make golint unhappy if the names' prefix is `owner`.